examples/example.krun

```diff
@@ -73,3 +73,9 @@ TEMP_READ_PAUSE = 1
 
 # CPU pinning (off by default)
 #ENABLE_PINNING = False
+
+# Lower and upper bound for acceptable APERF/MPERF ratios
+AMPERF_RATIO_BOUNDS = 0.97, 1.03
```

Comment:
Probably, although for random people's machines, we probably shouldn't set them *too* tight because they may just end up looping forever. Maybe 0.0985 and 1.1015 or something like that?